lib/pages/theme_page_settings.php

```diff
@@ -131,6 +131,21 @@ public static function install()
                 $status = rex_dir::create(rex_path::base($theme_folder.'/private/lib'));
             }
             if ($status) {
+                $status = rex_dir::create(rex_path::base($theme_folder.'/private/lib/yform'));
+            }
+            if ($status) {
+                $status = rex_dir::create(rex_path::base($theme_folder.'/private/lib/yform/action'));
+            }
+            if ($status) {
+                $status = rex_dir::create(rex_path::base($theme_folder.'/private/lib/yform/validate'));
+            }
+            if ($status) {
+                $status = rex_dir::create(rex_path::base($theme_folder.'/private/lib/yform/value'));
+            }
+            if ($status) {
```

Comment:
2. Diesen Ordner würde ich nicht innerhalb des lib-Ordners platzieren, sondern parallel zu den fragments, da er keine Klassen enthält.

Reply:
Da kommen die yform Klassen rein. Wenn der ausserhalb von lib liegt, muss man die zu Fuß laden.

Reply:
Ich meine nur den ytemplates-Ordner, der muss ja sowieso extra eingebunden werden. Die übrigen yform-Ordner sind schon richtig in lib, da hast du Recht.

Reply:
Ach jetzt raff ich das. Dein Kommentar steht immer oberhalb der Zeilen, um die es geht :)